src/display/api.js

```diff
@@ -903,6 +903,14 @@ class PDFDocumentProxy {
     return this._transport.getFieldObjects();
   }
 
+  /**
+   * @returns {Promise<Boolean>} A promise that is resolved with true if
+   * there are some actions in JavaScript.
+   */
+  hasJSActions() {
+    return this._transport.hasJSActions();
+  }
+
```

Comment:
It *looks* like this method is now completely unused/untested in the latest version of the patch.

Do we actually need to add this to the API, or could we instead wait until we see if there's an actual use for this? Generally speaking, it feels best to avoid adding unused things to the official API since we then have to maintain something that's unused/untested.

Hence I'd suggest reverting the changes in both the `src/display/api.js` and `src/core/worker.js` files for now.